core/templates/pages/exploration-player-page/new-lesson-player/new-lesson-player-components/player-sidebar.component.html

```diff
@@ -1,4 +1,89 @@
-<div class="oppia-lesson-player-sidebar"
-     [class.expanded]="isExpanded"
-     (click)="toggleSidebar()">
+<div class="oppia-lesson-player-sidebar">
+  <div class="player-desktop-sidebar-menu"
+       [class.expanded]="isExpanded">
+    <div class="lesson-player-sidebar-top">
+      <button class="player-sidebar-toggle"
+              (click)="toggleSidebar()"
+              [class.expanded]="isExpanded">
+        <div *ngIf="!isExpanded">
+          Open options
```

Comment:
All learner-facing strings should be i18n'd -- update assets/en,json and qqq.json and use the i18n codes here.

Reply:
Done